src/qbindiff/loader/backend/quokka.py

```diff
@@ -155,28 +158,9 @@ def value(self) -> int | None:
     @property
     def type(self) -> OperandType:
         """Returns the capstone operand type"""
-        op = self.cs_operand
-        typ = OperandType.unknown
-        cs_op_type = self.cs_operand.type
-
-        if cs_op_type == capstone.CS_OP_REG:
-            return OperandType.register
-        elif cs_op_type == capstone.CS_OP_IMM:
-            return OperandType.immediate
-        elif cs_op_type == capstone.CS_OP_MEM:
-            # A displacement is represented as [reg+hex] (example : [rdi+0x1234])
-            # Then, base (reg) and disp (hex) should be different of 0
-            if op.mem.base != 0 and op.mem.disp != 0:
-                typ = OperandType.displacement
-            # A phrase is represented as [reg1 + reg2] (example : [rdi + eax])
-            # Then, base (reg1) and index (reg2) should be different of 0
-            if op.mem.base != 0 and op.mem.index != 0:
-                typ = OperandType.phrase
-            if op.mem.disp != 0:
-                typ = OperandType.displacement
-        else:
-            raise NotImplementedError(f"Unrecognized capstone type {cs_op_type}")
-        return typ
+        # Get capstone type
+        cs = get_capstone_context(self.program().qb_prog.arch, self.program().qb_prog.endianness)
+        return convert_operand_type(cs.arch, self.cs_operand)
```

Comment:
```suggestion
        return convert_operand_type(self.program().qb_prog.capstone.arch, self.cs_operand)
```

In fact you can access the `capstone.Cs` object from the quokka `Program`